### PR TITLE
[JENKINS-35018] Migrate to new parent POM

### PR DESCRIPTION
**Pull request by andresrc**
See [JENKINS-35018](https://issues.jenkins-ci.org/browse/JENKINS-35018)

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,17 +30,22 @@
     <tag>HEAD</tag>
   </scm>
 
+  <properties>
+    <jenkins.version>1.580.3</jenkins.version>
```
Comment (andresrc) [inline pom.xml]: Baseline bump because of SECURITY-144

**Comment by oleg-nenashev** [comment]
🐝

**Comment by kwhetstone** [comment]
:bee:

**Comment by kwhetstone** [comment]
Is it possible to get this PR merged?  It helps with testing.

File: src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprintFacet.java [hunk 2]
```diff
@@ -30,12 +30,12 @@
  * Abstract class, which indicates {@link FingerprintFacet}s related to Docker.
  * This class is being used to retrieve the Docker-related data from {@link Fingerprint}s.
  * 
- * <p/>
+ * <p></p>
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFingerprintFacet.java]: :ant: `<p>` is more correct

**Comment by jglick** [comment]
:bee:

**Pull request merged**